### DISCUSSION TITLE
Maintainers?

### DISCUSSION THREAD

**Discussion by antoninu** [tomvictor/iotcore#49] [General]
Hi.

Is anybody maintaining this project at the moment? I see that the last commit was almost a year ago.

I find the project extremely useful for something I'm working on. I would be willing to help maintaining along somebody more experienced in MQTT.

**Comment by tomvictor** [comment]
Hi @antoninu, I am happy to maintain the project. But due to some extra workload I was not able to spend time on it.
Can you help me to collaborate  and develop?

Thanks

**Comment by bbartling** [comment]
Hi I am interested in this project too. I am familiar with Python learning both Django & MQTT. Can client MQTT device data be viewable through a Django web dashboard by chance with this package? 

And/or could one write topics to be published on the MQTT message bus for client devices to pickup? For example I am desiring some sort of a platform that can act as the MQTT broker/server that also has some sort of a web dashboard to view a client MQTT sensor data. I think it would be also cool is a web dashboard would have some sort of a text box where a user could enter in a setpoint or schedule for the remote MQTT client device to control too. Hopefully that makes sense! For example, home automation Django-iot dashboard the home owner can enter in a schedule to turn a light bulb on (remote MQTT client with a relay) and maybe view some other sensor data through the dashboard like room temperatures, etc...

Thanks for any response! I am a bit new to home automation but I have lots of work experience in commercial/hospital building automation BACnet type systems....
